druid-shell/src/platform/windows/application.rs

```diff
@@ -105,6 +104,11 @@ impl Application {
         unsafe {
             // Handle windows messages
             loop {
+                if let Ok(state) = self.state.try_borrow() {
```

Comment:
My experience is that putting *any* additional functionality in the runloop is problematic. The logic doesn't get invoked when the system is using its own runloop, which happens on live resize and also while a file dialog is open. Also, if and when we ever run as a guest (for example as a VST plugin), then we'll also be at the mercy of the host's runloop. But if we use the function sparingly, maybe we're ok.

Also, I think the name "DS_HANDLE_DROPPED" is confusing and should be changed to "BLOCKED" to be consistent with other functions.

See below where I suggest a comment to add.

Reply:
I wasnt to happy about this aswell. Given that `GetMessage` will block until a message is received its not guaranteed that the deferred messages are run immediately, and like you said on things like live resize the message is handled internally and our loop dont run. But what would be a good solution here? If we where to run a loop in a separate thread or use PeekMessage druid would waste/use a lot more cpu time. Maybe we could send a message whenever something is added to DeferredOps as to trigger the runloop?